docs/modules/ROOT/pages/usage.adoc

```diff
@@ -45,10 +46,11 @@ spec:
   image:
     productVersion: 3.3.1
     stackableVersion: 0.3.0
-  zookeeperConfigMapName: simple-kafka-znode
-  opa:
-    configMapName: simple-opa
-    package: kafka
+  clusterConfig:
+    zookeeperConfigMapName: simple-kafka-znode
+    opa:
+      configMapName: simple-opa
+      package: kafka
```

Comment:
In https://github.com/stackabletech/issues/issues/293 we decided to put `opa` below `authorization`. Would you agree to the change? Btw: We don't have opa tests for Kafka, created https://github.com/stackabletech/kafka-operator/issues/534
```suggestion
    authorization:
      opa:
        configMapName: simple-opa
        package: kafka
```

Reply:
Yeah done thats better. There is not much going on in terms of OPA for our Kafka now because of missing authentication...but yeah tests would be good here as well, but not part of this ticket.